.gitignore.orig

```diff
@@ -0,0 +1,48 @@
+*.py[cod]
```

Comment:
Ignires built python files. .pyc .pyo .pyd

On Fri, 9 Sep 2016 at 11:29 Gudni Karl Rosenkjaer <notifications@github.com>
wrote:

> In .gitignore.orig
> <https://github.com/simpeg/simpeg/pull/423#discussion_r78217723>:
>
> > @@ -0,0 +1,48 @@
> > +*.py[cod]
>
> What is this?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/simpeg/simpeg/pull/423/files/a298fb420d2866afd5bacec46f7b049eeff07e98#r78217723>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA3vYT_3f_pg56l5_S0eatgclAMf17yGks5qoZdkgaJpZM4J2F-i>
> .
>